ward/terminal.py

```diff
@@ -387,8 +399,8 @@ def output_all_test_results(
             console.print()
         return all_results
 
-    def print_divider(self):
-        console.print(Rule(style="muted"))
+    def print_divider(self) -> None:
+        print_divider()
```

Comment:
Any reason in particular for moving this out? The function only seems to be used here.